### ISSUE TITLE
Time out

### ISSUE THREAD

**Issue by taston** [bradjasper/ImportJSON#58]
Hi,

Very often I get error "504" on row 212:

> <html><body><h1>504 Gateway Time-out</h1> The server didn't respond in time. </body></html> 

How I can change connection-timeout value to for example 2000ms?

**Comment by taston** [comment]
Now i see:
https://developers.google.com/apps-script/reference/url-fetch/url-fetch-app

there is no option to set timeout...

**Comment by klassenjm** [comment]
@taston Did you ever come up with a solution for your timeout problem. I would like to use this script to pull data from a REST API, but the response in my current case can take a few minutes to return.

**Comment by taston** [comment]
No. I made button, if I see lag then I push button and send request again. That solution works in my case.
